src/Services/Processor/Substrate/Events/Implementations/FuelTanks/FreezeStateMutated.php

```diff
@@ -25,7 +25,7 @@ public function run(): void
         // Fail if it doesn't find the fuel tank
         $fuelTank = $this->getFuelTank($this->event->tankId);
 
-        if (is_null($this->event->ruleSetId)) {
+        if (empty($this->event->ruleSetId)) {
```

Comment:
We allow `ruleSetId` 0 right?

Reply:
correct, thanks

Reply:
We need to fix this right cause previously it only allow null